### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by anandsr21harshit**
Created 3 types of avatar

### CODE REVIEW

**Comment by suryaraj1** [comment]
Looks fine to me other than the few comments in CSS code. IMO can be merged. :smiley:

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    height: 0.8rem;
+    background-color: green;
+    border-radius: 50%;
+    position:absolute;
+    right: 25px;
+    top:75px;
+}
+
+.brb{
+    /* display: inline-block; */
```
Comment (Logan1x) [inline Components/Avatar/avatar.css]: this can be removed.

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    height: 0.8rem;
+    background-color: red;
+    border-radius: 50%;
+    position:absolute;
+    right: 25px;
+    top:75px;
+}
+
+.available{
+    /* display: inline-block; */
```
Comment (Logan1x) [inline Components/Avatar/avatar.css]: this too.

**Review by Logan1x — APPROVED** [review]
Looks good, as pointed out by others, unnecessary comments can be removed.

**Review by suryaraj1 — APPROVED** [review]

**Pull request merged**